### PR TITLE
Add double click to open cfn stack

### PR DESCRIPTION
**Pull request by hunterwerlla**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] Local run of `gradlew check` succeeds
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] A short description of the change has been added to the **CHANGELOG**

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .changes/next-release/bugfix-a462f2ce-093b-44ad-bf57-d506e715121b.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Fix double click opening Cloud Formation stack viewer"
```
Comment (abrooksv) [inline .changes/next-release/bugfix-a462f2ce-093b-44ad-bf57-d506e715121b.json:3]: This sounds like we the opposite of what we did.

Fix double click not*

or 

Enable double click*

Comment (hunterwerlla) [reply .changes/next-release/bugfix-a462f2ce-093b-44ad-bf57-d506e715121b.json]: Makes sense, reworded

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/Stack.kt [hunk 2]
```diff
@@ -37,17 +38,17 @@ class StackWindowManager(private val project: Project) {
     fun openStack(stackName: String, stackId: String) {
         assert(SwingUtilities.isEventDispatchThread())
         toolWindow.find(stackId)?.run { show() } ?: StackUI(project, stackName, stackId, toolWindow).start()
+        CloudformationTelemetry.open(project)
```
Comment (kiiadi) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/Stack.kt:41]: nice!

Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/Stack.kt:41]: oh it was already there you just moved it - I see.

**Pull request merged**